content/en/real_user_monitoring/guide/alerting-with-rum.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Alerting with RUM Data
+kind: guide
+description: Guide for creating alerts on RUM events.
+further_reading:
+    - link: '/real_user_monitoring/dashboards/'
+      tag: 'Documentation'
+      text: 'RUM Dashboards'
+    - link: '/monitors/create/types/real_user_monitoring/'
+      tag: 'Documentation'
+      text: 'RUM Monitor'
+---
+
+## Overview
+
+Real User Monitoring allows you to create alerts which notify you about anomalous behavior in your applications. You can create RUM monitors with complex conditions, predefined thresholds, and multiple queries to calculate averages, ratios, and performance indicator metrics (such as Apdex).
+
+## Define your search query
+
+To create a RUM monitor, see the [RUM Monitor documentation][1] first. You can add one or many queries to filter through your RUM data in the [RUM Explorer][2]. For each query, you can scope it on an application level or a more granular level like a specific page.
+
+You can use any facets that RUM collects, including [custom facets and measures][3]. Use the `measure by` field to measure view and action-related timings such as load time, time spent, and error count.
```

Comment:
Instead of action related timings let's go "counts"

Reply:
```suggestion
You can use any facets that RUM collects, including [custom facets and measures][3]. Use the `measure by` field to measure view-related counts such as load time, time spent, and error count.
```